ui/src/store/authSlice.ts

```diff
@@ -0,0 +1,27 @@
+// Package import
+import { createSlice } from '@reduxjs/toolkit';
+
+interface AuthState {
```

Comment:
Add a comment explaining your interface.

Reply:
That file is not related to this PR and is not optimized yet! So, I deleted it